chaincfg/params.go

```diff
@@ -476,6 +492,10 @@ var SimNetParams = Params{
 	// Mempool parameters
 	RelayNonStdTxs: true,
 
+	// Human-readable part for Bech32 encoded segwit addresses, as defined in
+	// BIP 173.
+	Bech32HRPSegwit: "tb", // always tb for test net
```

Comment:
I guess you are referring to `SimNetParams`? Just pushed the requested change there :)